templates/front/frontLayout.html.twig

```diff
@@ -0,0 +1,33 @@
+{% extends 'base.html.twig' %}
+
+{% block navbar %}
+<nav class="navbar navbar-expand navbar-light bg-light fixed-top" role="navigation">
+    <a class="navbar-brand" href="{{path('product_vehicles')}}">Mobilean</a>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('product_vehicles')}}">Nos solutions clés en main</a>
+    </div>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('product_chargingStations')}}">Nos solutions de ravitaillement</a>
+    </div>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('contact_estimate')}}">Je décris mon besoin</a>
+    </div>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('contact_partner')}}">Devenir installateur</a>
+    </div>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('information_about')}}">À propos de nous</a>
+    </div>
+</nav>
+{% endblock %}
+
+{% block footer %}
+<footer class="page-footer fixed-bottom bg-light">
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('contact_contact')}}">Contactez-nous</a>
+    </div>
+    <div class="nav-item">
+        <a class="nav-link" href="{{path('information_legal')}}">Mentions légales</a>
+    </div>
```

Comment:
a div nav-item should probably be a nav containing a list of links

Reply:
Nav has been created using bootstrap, hence all the divs. Using a list would need more css and would negate the use of bootstrap.